### PR TITLE
Postsubmit for deploying boskos and updating boskos config

### PR DESCRIPTION
**Pull request by clarketm**
Postsubmit(s) to deploy boskos and update boskos config.

/hold
> ~~`prow-deployer-service-account` needs to be updated **before** this can be merged.~~

**Update:**
`prow-deployer-service-account` updated with `container.admin` permissions.

### CODE REVIEW

File: prow/cluster/jobs/istio/test-infra/istio.test-infra.trusted.master.yaml [hunk 1]
```diff
@@ -29,6 +29,58 @@ postsubmits:
...
+      testgrid-create-test-group: "false"
+    spec:
+      containers:
+        - image: gcr.io/k8s-testimages/gcloud-in-go:v20190125-cc5d6ecff3
+          command:
+            - bash
+          args:
+            - -c
+            - |
+              gcloud auth activate-service-account --key-file=/creds/service-account.json && \
```
Comment (fejta) [inline prow/cluster/jobs/istio/test-infra/istio.test-infra.trusted.master.yaml:48]: Consider trying to make the command just `make -C boskos boskos-config` (aka skip bash) and set the `env: GOOGLE_APPLICATION_CREDENTIALS=/creds/service-account.json`, which may or may not work

**Review by fejta — APPROVED** [review]
/hold

File: prow/cluster/jobs/istio/test-infra/istio.test-infra.trusted.master.yaml (hunk 1 above)
Comment (clarketm) [reply prow/cluster/jobs/istio/test-infra/istio.test-infra.trusted.master.yaml:48]: Okay, Let me give that a shot.

Comment (clarketm) [reply prow/cluster/jobs/istio/test-infra/istio.test-infra.trusted.master.yaml:48]: It did not seem to work. Maybe it has to do with the fact `make` runs in subshells ... not sure. I would like to get this simplified but can do this in a follow-up.

Comment (clarketm) [reply prow/cluster/jobs/istio/test-infra/istio.test-infra.trusted.master.yaml:48]: #1794 is a follow-up attempt to clean this up.

**Pull request merged**